### ISSUE TITLE
ExchangeId Identity Token: Firewall rules for the meta data server

### ISSUE THREAD

**Issue by pickartzu** [OfficeDev/outlook-dev-docs#275]
For the verification of the Exchange identity token, we have to implement firewall rules in order to access the server of the URL from the meta data endpoint ("appctx".” amurl”).
Can you give us a hint which server can be possible in order to implement the firewall rules for them?
In a first step it would be helpful to get an answer for this question at least for O365 domains and outlook.com.

**Comment by jasonjoh** [comment]
@lindalu-MSFT @ElizabethSamuel-MSFT

**Comment by lindalu-MSFT** [comment]
@jasonjoh  I assigned to two of the writers on the Add-ins team. Do you know what we need to change in the topic? If you can provide any guidance, it would be greatly appreciated. Thanks in advance.

**Comment by lindalu-MSFT** [comment]
@ElizabethSamuel-MSFT @AlexJerabek I assigned this to you to update the topic.

**Comment by jasonjoh** [comment]
@lindalu-MSFT No sorry, I have no idea. I assume all that's needed are the servers for Office 365 (assuming they are fixed endpoints). You might start with @wamaina to see what info we can provide here.

**Comment by lindalu-MSFT** [comment]
I unassigned @AlexJerabek.  @ElizabethSamuel-MSFT to research technical details to follow-up with this. THANKS. :)

**Comment by ElizabethSamuel-MSFT** [comment]
@pickartzu: Here is an article about [Office 365 URLs and IP address ranges](https://docs.microsoft.com/office365/enterprise/urls-and-ip-address-ranges). This support article ([Office 365 endpoint reference](https://support.office.com/en-us/article/Office-365-endpoints-reference-a9ca90db-7263-4234-a881-d2b9bb769118)) may also be useful.

**Comment by ElizabethSamuel-MSFT** [comment]
@pickartzu: If you're still blocked by this issue, please reactivate or create a new issue. Thanks.
